### PR TITLE
Bump deps and minor updates to docs

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: hw/hw.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+The NervesKey is a tiny circuit board with an ATECC608A cryptographic chip from
+Microchip Technology. It is an inexpensive addition to the Raspberry Pi and
+other embedded computing platforms that include an I2C bus.
+
+![NervesKey assembled](assets/assembled.jpg "Assembled NervesKey")
+
+The production NervesKey board contains the tiny 'UDFN' version of the part
+[ATECC608A-MAHDA](https://www.digikey.com/product-detail/en/microchip-technology/ATECC608A-MAHDA-S/ATECC608A-MAHDA-STR-ND/7928113)
+and a 0.1uF 0402 size capacitor.  These boards are not easy assembled, so they
```
Comment (jjcarstens) [inline hw/hw.md]: Should it be “not easily assembled”?

File: hw/hw.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Hardware
+
+The NervesKey is a tiny circuit board with an ATECC608A cryptographic chip from
+Microchip Technology. It is an inexpensive addition to the Raspberry Pi and
```
Comment (jjcarstens) [inline hw/hw.md]: Is it worth linking to `Microchip Technology`? I’m just assuming this is a company or brand

File: hw/hw.md (hunk 1 above)
Comment (fhunleth) [reply hw/hw.md]: Fixed

File: hw/hw.md (hunk 2 above)
Comment (fhunleth) [reply hw/hw.md]: Fixed.

**Comment by fhunleth** [comment]
I'm wanting to rewrite this a little bit more, but it seemed incrementally better than what was in the repo.

**Pull request merged**